src/clients/inst_autoinit.rb

```diff
@@ -202,6 +202,7 @@ def processProfile
       AutoinstGeneral.Import(Profile.current.fetch("general",{}))
       AutoinstGeneral.SetSignatureHandling
       AutoinstGeneral.SetMultipathing
+      AutoinstGeneral.set_btrfs_default_subvolume_name
```

Comment:
This looks like a layering violation. Can we move the multipathing and btrfs init to some `storage_autoinit` method?